rules/anti-static/obfuscation/js.yara

```diff
@@ -3,9 +3,9 @@ import "math"
 private rule probably_js {
   strings:
     $f_function = /function\(\w{0,8}\)/
-    $f_const    = "const" fullword
-    $f_return   = "return" fullword
-    $f_var      = "var" fullword
+    $f_const    = /\bconst\b/
```

Comment:
looking at const/return/var, I think the strictest you can safely go is `\bword\s`; how about that?

Reply:
Updated in [`4260efd` (#694)](https://github.com/chainguard-dev/malcontent/pull/694/commits/4260efda23707a4e7d4a8539902b41a090f16684).